src/api_server/src/request/mmds.rs

```diff
@@ -5,11 +5,21 @@ use crate::parsed_request::{Error, ParsedRequest, RequestAction};
 use crate::request::Body;
 use logger::{IncMetric, METRICS};
 use micro_http::StatusCode;
-use vmm::rpc_interface::VmmAction::SetMmdsConfiguration;
+use vmm::rpc_interface::VmmAction::{GetMmdsConfiguration, SetMmdsConfiguration};
 
-pub(crate) fn parse_get_mmds() -> Result<ParsedRequest, Error> {
+pub(crate) fn parse_get_mmds(path_second_token: Option<&&str>) -> Result<ParsedRequest, Error> {
     METRICS.get_api_requests.mmds_count.inc();
-    Ok(ParsedRequest::new(RequestAction::GetMMDS))
+    match path_second_token {
+        None => Ok(ParsedRequest::new(RequestAction::GetMMDS)),
+        Some(&"config") => Ok(ParsedRequest::new_sync(GetMmdsConfiguration)),
+        Some(&unrecognized) => {
```

Comment:
I would have this as `_ => {...` because it is clearer that it's an unsupported case.